src/catalog/catalog_entry/scalar_function_catalog_entry.cpp

```diff
@@ -5,10 +5,6 @@
 namespace kuzu {
 namespace catalog {
 
-ScalarFunctionCatalogEntry::ScalarFunctionCatalogEntry(
-    const char* name, function::function_set functionSet)
-    : ScalarFunctionCatalogEntry{std::string{name}, std::move(functionSet)} {}
-
 ScalarFunctionCatalogEntry::ScalarFunctionCatalogEntry(
```

Comment:
i think we can remove scalarFunctionCatalogEntry,aggregateFunctionCatalogEntry,TableFunctionCatalogEntry now?

Reply:
I'll check and remove it if it hasn't been used in other part of the code.